model/project_ref.go

```diff
@@ -404,6 +410,49 @@ func (p *ProjectRef) GetPatchTriggerAlias(aliasName string) (patch.PatchTriggerD
 	return patch.PatchTriggerDefinition{}, false
 }
 
+func (p *ProjectRef) GetMergedConfig(version string) *MergedProjectConfig {
```

Comment:
I'm not sure about making another new struct, especially since so many things will be shared between them. Could we instead destructively modify p (with a function like IncludeParserProject or something)? Or return a new merged project ref that incorporates the ParserProject version of the variables.